java/src/main/java/com/ncc/aif/ValidateAIF.java

```diff
@@ -59,6 +59,7 @@
     private static final String AIDA_SHACL_RESNAME = AIF_ROOT + "aida_ontology.shacl";
     private static final String NIST_SHACL_RESNAME = AIF_ROOT + "restricted_aif.shacl";
     private static final String NIST_HYPOTHESIS_SHACL_RESNAME = AIF_ROOT + "restricted_hypothesis_aif.shacl";
```

Comment:
Is this hypothesis variable still being used?

Reply:
removed